src/Common/src/System/Xml/XmlConvertEx.cs

```diff
@@ -66,7 +66,7 @@ public static string[] SplitString(string value)
 
         public static Uri ToUri(string s)
         {
-            if (s != null && s.Length > 0)
```

Comment:
We aren't using C#6 features as of right now, but that looks like it should be equivalent.